<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTaskFormWeekly</class>
 <widget class="QDialog" name="AddTaskFormWeekly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Task</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>480</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPlainTextEdit" name="taskDesc">
         <property name="placeholderText">
          <string>Add description</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDateTimeEdit" name="dateTimeEnd"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QPushButton" name="pbSaveTask">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDateTimeEdit" name="dateTimeStart"/>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="taskTitle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="maxLength">
          <number>1024</number>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Add title</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>All day</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Does not repeat</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Daily</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Weekly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Monthly</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Annually</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Priority: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>*</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>**</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>***</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>****</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>*****</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
